csgo_bot.py

```diff
@@ -1,12 +1,9 @@
-# import pdb
-import sys
+﻿import sys
 import os
 from importlib import reload
-
 import telebot
-
 from db_helper import DBHelper
-
+from telebot import types
```

Comment:
According to [1]  we should leave the structure like:

import sys
import os
from importlib import reload

import telebot
from telebot import types (why not using directly telebot.types in your calls?)

from db_helper import DBHelper


[1] https://stackoverflow.com/questions/22722976/import-order-coding-standard